randovania/cli/prime_database.py

```diff
@@ -570,7 +601,6 @@ def create_subparsers(sub_parsers):
         "--game",
         type=str,
         choices=[game.value for game in iterate_enum(RandovaniaGame)],
-        default=RandovaniaGame.METROID_PRIME_ECHOES.value,
```

Comment:
@henriquegemignani Will this break anything? In hindsight, I don't think I needed to make this change.

Reply:
I don't really care about the stability of the cli, change freely.